src/couch/src/couch_auth_lockout.erl

```diff
@@ -0,0 +1,85 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License.  You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-module(couch_auth_lockout).
+-include_lib("couch/include/couch_db.hrl").
+
+-define(LRU, couch_lockout_lru).
+
+-export([is_locked_out/3, lockout/3]).
+
+is_locked_out(#httpd{} = Req, UserName, UserSalt) ->
+    case lockout_mode() of
+        off ->
+            false;
+        Mode ->
+            is_locked_out_int(Req, Mode, UserName, UserSalt)
```

Comment:
Minor suggestion: short clauses like these can be formatted inline  `off -> false;` and `Mode -> ...`

Reply:
I'm not a fan of that style tbh, a shame our formatter isn't opinionated about it.

Reply:
That's fine, if you think it looks good. It's just style suggestion and erlfmt likes it either way.